src/prefect/environments/execution/fargate/fargate_task.py

```diff
@@ -286,8 +286,8 @@ def execute(self, flow: "Flow") -> None:  # type: ignore
                 "environment": [
                     {
                         "name": "PREFECT__CLOUD__AUTH_TOKEN",
-                        "value": config.cloud.agent.auth_token
-                        or config.cloud.auth_token,
+                        "value": config.cloud.agent.get("auth_token")
+                        or config.cloud.get("auth_token"),
```

Comment:
dumb question but any issue with `None`s in this template?

Reply:
Not that I am aware of but it wouldn't hurt to make it empty strings, will update